content/python/concepts/random-module/terms/triangular/triangular.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: 'triangular'
+Description: 'Returns a random floating number from a triangular distribution.'
+Subjects: 
+  -"Computer Science"
+  -"Data Science"
+Tags:
+  -"Functions"
+  -"Methods"
+  -"Random"
+Catalog Content: 
+  -"learn-python-3"
+  -"paths/computer-science"
+  -"paths/data-science"
+---
+
+"The ** `.triangular()` ** method returns a random floating number from a triangular distribution. Input parameters are `low`, `high` and `mode`. If `mode` not provided, the midpoint is set to default."
+
+## Syntax
+
+```pseudo
+random.triangular(low,high,mode)
+```
+
+Where `low` and `high` correspond to the required lower and upper limits of distribution, and `mode` is an optional input for peak of the distribution.
```

Comment:
Instead of this, could you define the parameters in the following format:

- `low`: description
- `high`: description
- `mode`: description